.github/workflow-templates/no_response.yaml

```diff
@@ -0,0 +1,33 @@
+# A workflow to close issues where the author hasn't responded to a request for
+# more information; see https://github.com/godofredoc/no-response for docs.
+
+name: No Response
+
+# Both `issue_comment` and `scheduled` event types are required.
+on:
+  issue_comment:
+    types: [created]
+  schedule:
+    # Every day at 8:00 am
+    - cron: '0 8 * * *'
+
+# All permissions not specified are set to 'none'.
+permissions:
+  issues: write
+
+jobs:
+  noResponse:
+    runs-on: ubuntu-latest
+    if: ${{ github.repository_owner == 'googlecolab' }}
+    steps:
+      - uses: godofredoc/no-response@0ce2dc0e63e1c7d2b87752ceed091f6d32c9df09
+        with:
+          responseRequiredLabel: "reply-needed"
+          responseRequiredColor: 8a9113
```

Comment:
I think it's 8B9213 ?  https://screenshot.googleplex.com/6Pk3aGRLebgvrpJ

Not sure if I'm doing this right though